.env.template

```diff
@@ -7,3 +7,4 @@ SECRET_KEY=secret-key
 TRELLO_API_KEY=trello-api-key
 TRELLO_TOKEN=trello-token
 TRELLO_BOARD_ID=trello-board-id
+TRELLO_ORGANIZATION_ID=trello-organization-id
```

Comment:
Will comment below but we could skip this unless you actually care which organisation it goes in - just not passing this argument and allowing it to hit your default personal organisation would probably be fine here.